### ISSUE TITLE
Is adversarial images used in distillation loss?

### ISSUE THREAD

**Issue by YangJae96** [dipamgoswami/ADC#2]
Hi. 

I have question about the method which I am confused.

In the paper, I can see that in Figure3, the generated adversarial images are fed to old model and estimates logit vector? and used in knowledge distillation loss. 

![image](https://github.com/dipamgoswami/ADC/assets/57695621/c8881ef3-a380-4fe6-b5a5-2377df20db50)

But in the above code, I can only see the perturbed images(x_, y_) is only used in updating the old prototype, which is illustrated in Section 3.3 in your paper, Drift Compensation!

Are the perturbed adversarial images not used with along new task data when training new model?

Thanks in advance.

**Comment by dipamgoswami** [comment]
Hi, 

The adversarial images are used only for the drift compensation (as denoted by the dotted blue lines in Fig. 3 from the paper). And only the new task images are used for the training (as denoted by the red lines in Fig. 3). The perturbed images are not used along with new task data for training.
